src/components/LayerCollection/LayerPO.js

```diff
@@ -52,6 +63,12 @@ class LayerPO extends Component {
             initialValues={order}
             onSubmit={this.updatePO}
           />
+          {updateOrderErrorReason &&
+          <UpdateOrderErrorModal
+            orderNumber={order.poNumber}
+            reason={updateOrderErrorReason}
```

Comment:
are you sure you need to show all errors inside modal?

Reply:
I've asked @aliaksei-chumakou about it, his answer was yes

Reply:
I've pointed you to the related ticket, and it contains only 2 error codes, related to inactive vendors.
https://issues.folio.org/browse/MODORDERS-170
let's keep other error codes as is - in notifications